GDrive/README.md

```diff
@@ -1,67 +1,60 @@
-# AGDroid GDrive: Basic Mobile Application showing the AeroGear Authz (OAuth2) feature on Android
----------
+# Mobile app showing the AeroGear Authz feature on Android
+
 Author: Daniel Passos (dpassos) & Summers Pittman (supittma)   
 Level: Beginner   
 Technologies: Java, Android   
 Summary: A basic example of OAuth2 : Login and retrieve data.   
-Target Product: -   
-Product Versions: -   
-Source: https://github.com/aerogear/aerogear-android-cookbook/tree/master/GDrive   
+Source: https://github.com/aerogear/aerogear-android-cookbook/tree/master/GDrive
 
 ## What is it?
 
-The ```AGDroid GDrive``` project demonstrates how to include Authz/OAuth2 functionality in Android applications.
+_GDrive_ project demonstrates how to include Authz/OAuth2 functionality in Android applications.
 
 This simple project consists of a ready-to-build Android application. Before building the application, you must create an OAuth2 credential in Google's console. The credentials must then be inserted into the application source code. After this is complete, the application can be built and deployed to Android devices.
 
 When the application is deployed to an Android device, the application will ask for your Google credentials, ask permission to access your account and retrieve your documents.
 
 ## How do I run it?
 
-### 1. System Requirements
-
-* [Java 7](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* [Gradle 2.1](http://www.gradle.org/)
-* Latest [Android SDK](https://developer.android.com/sdk/index.html) and [Platform version 23](http://developer.android.com/tools/revisions/platforms.html)
+### System Requirements
 
-For the complete instructions about how to setup Google credentials, visit our [OAuth2 documentation guide](https://aerogear.org/docs/guides/security/oauth2-guide/#Google)
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Gradle](http://www.gradle.org/)
```

Comment:
The README.md says `gradle clean build`, without the w, that's the one I used when testing the Two-factor PR and failed